docs/source/json-api/index.rst

```diff
@@ -546,6 +546,111 @@ HTTP Response
 
 Formatted similar to :ref:`Exercise by Contract ID response <exercise-response>`.
 
+Create and Exercise in the Same Transaction
+*******************************************
+
+This command allows creating a contract and exercising a choice on the newly created contract in the same transaction.
+
+HTTP Request
+============
+
+- URL: ``/v1/create-and-exercise``
+- Method: ``POST``
+- Content-Type: ``application/json``
+- Content:
+
+.. code-block:: json
+
+    {
+      "templateId": "Iou:Iou",
+      "payload": {
+        "observers": [],
+        "issuer": "Alice",
+        "amount": "999.99",
+        "currency": "USD",
+        "owner": "Alice"
+      },
+      "choice": "Iou_Transfer",
+      "argument": {
+        "newOwner": "Bob"
+      }
+    }
+
+Where:
+
+- ``templateId`` -- the initial contract template identifier, in the same format as in same as in the :ref:`create request <create-request>`,
+- ``payload`` -- the initial contract fields as defined in the DAML template and formatted according to :doc:`lf-value-specification`,
+- ``choice`` -- DAML contract choice, that is being exercised,
+- ``argument`` -- contract choice argument(s).
+
+HTTP Response
+=============
+
+Please note that the response below is for a consuming choice, so it contains:
+
+- ``created`` and ``archived`` events for the initial contract (``"contractId": "#1:0"``), which was created and archived right away when a consuming choice was exercised on it,
+- a ``created`` event for the contract that is the result of the choice exercise (``"contractId": "#1:2"``).
```

Comment:
@hurryabit are you sure you want this? The original contract created and archived, we do not conflate like we do in the streaming API.

#4507 says:
> Response format: exactly the resulting transaction tree you get from ledger-API in submit-and-wait, formatted exactly as for the exercise endpoint.

Reply:
When using the streaming API, one is more interested in getting "actionable" contracts. For `create-and-exercise` I think it can make sense to return the response as is.
For the response to a regular `exercise` command we don't do any conflation of transient contracts either.

Reply:
all right... leaving it as is.

Reply:
I prefer to _not_ conflate here. So, all good.